### DISCUSSION TITLE
while connecting to db facing below, can anybody have solution for this

### DISCUSSION THREAD

**Discussion by karluke-akash** [prisma/prisma#20172] [Q&A]
### Bug description

Error: request to https://binaries.prisma.sh/all_commits/4bc8b6e1b66cb932731fb1bdbbc550d1e010de81/windows/query_engine.dll.node.gz.sha256 failed, reason: unable to get local issuer certificate

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by nurul3101** [comment]
Hello @karluke-akash 👋 

Can you please let us know which OS and Prisma version are you using?

I would also recommend changing your network to see if you still get the same error in other network. This issue might occur due to a network firewall.

**Reply by nurul3101** [reply]
@karluke-akash Please let us know in case you still need any assistance.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
